.gitignore

```diff
@@ -69,6 +69,7 @@ __pycache__/
 /doc
 /inst/
 /llvm/
+/mir_dump
```

Comment:
I wind up with `mir_dump` directories scattered al over (e.g., often in `src/test/ui`) -- maybe we can make this a more general case? I forget what leading `/` means though.

Reply:
IIRC, the `/` just makes it match from the root of the repository. I've removed the `/` so it should match anywhere in the tree.